sdk/monitor/azure-monitor-query/src/test/java/com/azure/monitor/query/LogsQueryClientTest.java

```diff
@@ -64,33 +63,34 @@ public class LogsQueryClientTest extends TestProxyTestBase {
 
     private String resourceId;
 
+    private TokenCredential credential;
+
     @BeforeEach
     public void setup() {
         workspaceId = getLogWorkspaceId(interceptorManager.isPlaybackMode());
         additionalWorkspaceId = getAdditionalLogWorkspaceId(interceptorManager.isPlaybackMode());
         resourceId = getLogResourceId(interceptorManager.isPlaybackMode());
+        credential = TestUtil.getTestTokenCredential(interceptorManager);
         LogsQueryClientBuilder clientBuilder = new LogsQueryClientBuilder()
-                .retryPolicy(new RetryPolicy(new RetryStrategy() {
-                    @Override
-                    public int getMaxRetries() {
-                        return 0;
-                    }
-
-                    @Override
-                    public Duration calculateRetryDelay(int i) {
-                        return null;
-                    }
-                }));
+            .credential(credential)
+            .retryPolicy(new RetryPolicy(new RetryStrategy() {
+                @Override
+                public int getMaxRetries() {
```

Comment:
Just curious - what are we setting maxRetries to 0? Is that the default?

Reply:
This was to avoid the server timeout tests to take a really long time exponentially backing off until we received the expected error code. However, that test has since been updated and we no longer have a forever loop to test the server behavior. So, this custom retry policy can be removed.